### ISSUE TITLE
[Feature Request] Allow tracking of additional measurments

### ISSUE THREAD

**Issue by AlexisOlson** [astashov/liftosaur#70]
I would like to be able to track additional metrics like

### Metabolic markers
* VO2 max
* Resting energy
* Walking heart rate average

### Recovery indicators
* Resting heart rate
* Resting respiratory rate
* Sleep
* Heart rate variability
* Cardio recovery
* Subjective scores (e.g. tiredness/soreness)

### Vital signs
* Blood pressure
* Body temperature

### Activity stats
* Steps
* Distance walked / run
* Stair flights climbed

These are just a subset of many possible measurements, so it might be worth allowing custom measurements so users aren't limited to a pre-defined list.

I know this is primarily a lifting app but as long as you aren't trying to sync these stats from somewhere else like Apple Health, I'd expect it to be a simple expansion of existing functionality. If you think this request is a good idea, I'd like to try writing a PR myself.

**Comment by astashov** [comment]
The feature kinda makes sense, but I'll have postpone it for now, to avoid complicating the transitioning to the new editor, sorry! Maybe later someday.

**Comment by astashov** [comment]
Moved it to the feature roadmap

**Comment by HarishHary** [comment]
hello @astashov, I noticed that the heart rate are only collected every 10min or so? and the calories spent are not fully accurate? Not sure if this is normal. Watch is working properly and correctly attached as well. I have used other apps in the past and the collection of those metrics were far more accurate (more data that what is being done here).

Love the app btw ! Pretty nice work ! Keep up the good!
